src/format.ts

```diff
@@ -100,11 +101,11 @@ function fixFormat(srcFiles: string[], baseArgs: string[]): Promise<boolean> {
  *
  * @param srcFiles list of source files
  */
-function checkFormat(srcFiles: string[], baseArgs: string[], options: Options): Promise<boolean> {
+async function checkFormat(
```

Comment:
I believe this function does not need to be `async`, since the use of `await` is in a nested function.

Reply:
oops, forgot to remove